velox/functions/sparksql/Bitwise.cpp

```diff
@@ -81,10 +84,47 @@ struct ShiftRightFunction {
   }
 };
 
+template <typename T>
+struct BitCountFunction {
+  template <typename TInput>
+  FOLLY_ALWAYS_INLINE void call(int32_t& result, TInput num) {
+    constexpr int kMaxBits = sizeof(TInput) * CHAR_BIT;
+    result = std::bitset<kMaxBits>(num).count();
+  }
+};
+
+template <typename T>
+struct BitGetFunction {
+  template <typename TInput>
+  FOLLY_ALWAYS_INLINE void call(int8_t& result, TInput num, int32_t pos) {
+    constexpr int kMaxBits = sizeof(TInput) * CHAR_BIT;
+    VELOX_USER_CHECK(
```

Comment:
Split this check into 2 separate checks for better error message. Remove 'pos' from the error message.

```
VELOX_USER_CHECK_GE(pos, 0, ...);
VELOX_USER_CHECK_LT(pos, kMaxBits, ...);
```